styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/MetricsStats.java

```diff
@@ -209,6 +214,15 @@ public void pullImageError() {
     pullImageErrorMeter.mark();
   }
 
+  @Override
+  public void registerResourceCount(String resource, Gauge<Long> resourceCount) {
+    registry.register(RESOURCE_USAGE.tagged("resource-total", resource), resourceCount);
```

Comment:
Semantically I think these could be just two different "what"s. There's some redundancy in a metric that says `"what": "resource-usage", "resource-used": 101`, and there's some contradiction in a metric that says `"what": "resource-*usage*", "resource-*configured*": 101`.